README.md

```diff
@@ -2,6 +2,52 @@
 
 This repository contains examples on how to integrate the [PACE Cloud SDK Capacitor plugin](https://github.com/pace/cloud-sdk-capacitor-plugin) with Ionic.
 
+## Android
+### Setup
+Install dependencies
+
+`npm install`
+
+Install the plugin locally
+
+`npm install ../../cloud-sdk-capacitor-plugin`
+
+Run
+
+`npm run build`
+
+Sync the plugin and open it in Android Studio:
+
+`npx cap sync`
+
+`npx cap open android`
```

Comment:
This section is not needed, right? It's the same as the `# Local Installation`

Reply:
I dont see `# Local Installation` section in the README.md

Reply:
Ah, it's in the Resume within the vue-starter directory.

I think we should replace this readme with the one in there.

Reply:
updated